packages/files-ui/src/Components/Modules/LoginModule/InitialScreen.tsx

```diff
@@ -171,6 +180,10 @@ const InitialScreen = ({ className }: IInitialScreen) => {
   const [error, setError] = useState<string | undefined>()
   const [errorEmail, setErrorEmail] = useState("")
   const maintenanceMode = process.env.REACT_APP_MAINTENANCE_MODE === "true"
+  const maintenanceWindowTimestamp = (process.env.REACT_APP_MAINTENANCE_TIMESTAMP &&
+    process.env.REACT_APP_MAINTENANCE_TIMESTAMP !== "")
+    ? Number(process.env.REACT_APP_MAINTENANCE_TIMESTAMP)
+    : undefined
```

Comment:
We can cast it to number, and check if it's falsy then later.
```suggestion
  const maintenanceWindowTimestamp = Number(process.env.REACT_APP_MAINTENANCE_TIMESTAMP)
```